package.json

```diff
@@ -94,7 +94,7 @@
     "facepaint": "^1.2.0",
     "hoist-non-react-statics": "^2.3.1",
     "lodash": "^4.17.4",
-    "markdown-to-jsx": "^6.6.1",
+    "markdown-to-jsx": "6.6.1",
```

Comment:
Why? Isn't this what we have a lockfile for? Did it not work?

Reply:
Sadly no. Didn't you have the same issue? I didn't want to commit the updated specs as I wasn't sure if they are from an old version or a new version with the additional ids.

We can put it back if we update the tests appropriately.